scripts/pylib/twister/twisterlib.py

```diff
@@ -2632,10 +2632,11 @@ def process(self, pipeline, done, message, lock, results):
                     self.instance.status = "passed"
                 pipeline.put({"op": "report", "test": self.instance})
             else:
+                # Here we check the runtime filter results coming from running cmake
```

Comment:
during build stage, there are also some cases skipped due to ram/rom overflow, do we need to set them as runtime filter too? this case is very rare, but it does exists.

Reply:
good point, those need to stay as skipped with a new type I guess, because potentially they were skipped only because of increased size, so they need some attention.